Remove GNU make-isms: $< is only valid in suffix rules.
Binaries will be stripped automatically.
--- Makefile.orig	Thu Nov  7 21:15:59 2013
+++ Makefile	Thu Nov  7 21:16:39 2013
@@ -14,11 +14,10 @@ OBJECTS := yeahconsole.o
 SOURCES := yeahconsole.c
 
 $(TARGET): $(OBJECTS) 
-	$(CC) $(DEFINES) $(INCLUDES) $(LIB_DIRS) $(LIBS) -o $@ $<
-	strip $@
+	$(CC) $(DEFINES) $(INCLUDES) $(LIB_DIRS) $(LIBS) -o $@ $?
 
 $(OBJECTS): $(SOURCES) 
-	$(CC) $(FLAGS)  $(DEFINES) $(INCLUDES) $(LIB_DIRS) -c -o $@ $<
+	$(CC) $(FLAGS)  $(DEFINES) $(INCLUDES) $(LIB_DIRS) -c -o $@ $?
 
 clean:
 	rm -rf $(TARGET) $(OBJECTS)
